spec/rubocop/cop/style/hash_each_methods_spec.rb

```diff
@@ -87,5 +87,29 @@
         expect_no_offenses('each_value { |v| p v }')
       end
     end
+
+    context "when `AllowedReceivers: ['execute']`" do
+      let(:cop_config) { { 'AllowedReceivers' => ['execute'] } }
+
+      it 'does not register an offense when receiver is `execute` method' do
+        expect_no_offenses(<<~RUBY)
+          execute(sql).values.each { |v| p v }
+        RUBY
+      end
+
+      it 'does not register an offense when receiver is `execute` variable' do
+        expect_no_offenses(<<~RUBY)
+          execute = do_something(argument)
+          execute.values.each { |v| p v }
+        RUBY
+      end
+
+      it 'does not register an offense when receiver is not allowed name' do
```

Comment:
`does not` seems to be a copy-paste typo

Reply:
Oops! Thanks!